cdap-cli/src/main/java/io/cdap/cdap/cli/CLIMain.java

```diff
@@ -81,10 +83,19 @@
     "h", "help", false, "Print the usage message.");
 
   private static final Option URI_OPTION = new Option(
-    "u", "uri", true, "CDAP instance URI to interact with in" +
+    "u", "uri", true, "(Deprecated. Please use option l instead). CDAP instance URI to interact with in" +
     " the format \"[http[s]://]<hostname>[:<port>[/<namespace>]]\"." +
     " Defaults to \"" + getDefaultURI().toString() + "\".");
 
+  private static final Option LINK_OPTION = new Option(
+    "l", "link", true, "CDAP instance URI to interact with in" +
+    " the format \"[http[s]://]<hostname>[:<port>]\" or \"[https[s]://]<hostname>[/<apipath>]\"." +
```

Comment:
There is no need to have or for the format. The format is just:

`http[s]://<hostname>[:<port>][/<path>]`

Reply:
Done